argoverse/utils/se3.py

```diff
@@ -55,20 +55,33 @@ def inverse(self) -> "SE3":
         """
         return SE3(rotation=self.rotation.T, translation=self.rotation.T.dot(-self.translation))
 
-    def right_multiply_with_se3(self, right_se3: "SE3") -> "SE3":
-        """Right multiply the transformation matrix with another SE3 instance.
+    def compose(self, right_se3: "SE3") -> "SE3":
+        """Compose (right multiply) this class' transformation matrix T with another SE3 instance.
```

Comment:
I might would say "by" instead of "with", but probably a matter of preference ...